server/routes/one-time-code-auth/RequestOneTimeCodeController.test.ts

```diff
@@ -36,11 +34,6 @@ describe('RequestOneTimeCodeController', () => {
     verifyOneTimeCodeController as unknown as VerifyOneTimeCodeController
   )
 
-  beforeEach(() => {
```

Comment:
`oneTimeCodeValidityDuration` and `lsjSessionDuration` are no longer part of the view 👇 so are not needed in this test